articles/how_to_work_with_large_language_models.md

```diff
@@ -6,14 +6,14 @@
 
 The magic of large language models is that by being trained to minimize this prediction error over vast quantities of text, the models end up learning concepts useful for these predictions. For example, they learn:
 
-* how to spell
-* how grammar works
-* how to paraphrase
-* how to answer questions
-* how to hold a conversation
-* how to write in many languages
-* how to code
-* etc.
+- how to spell
+- how grammar works
+- how to paraphrase
+- how to answer questions
+- how to hold a conversation
+- how to write in many languages
+- how to code
+- etc.
```

Comment:
16. how to bring data together
17. speech recognition
18. handwriting recognition
19. summarize articles
20. machine translation